environments/production/aws-resources.yml

```diff
@@ -280,6 +280,8 @@ pgshard4-production: pgshard4-production.cl9dmuo3ok4h.us-east-1.rds.amazonaws.co
 pgshard5-production: pgshard5-production.cl9dmuo3ok4h.us-east-1.rds.amazonaws.com
 pgsynclog0-production: pgsynclog0-production.cl9dmuo3ok4h.us-east-1.rds.amazonaws.com
 pgucr0-production: pgucr0-production.cl9dmuo3ok4h.us-east-1.rds.amazonaws.com
+pillow1-production: 10.202.12.128
+pillow1-production.instance_id: i-0354a00e5ace39eb2
```

Comment:
@calellowitz you asked about this. I'll remove it. It was added automatically by pulling the list of current resources from AWS.